downstream/snippets/technology-preview.adoc

```diff
@@ -0,0 +1,3 @@
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
This is a recommendation, so feel free to use/refuse as you deem fit:
I'd prefer having the 3rd statement at the start of the paragraph, and the current 1st and 2nd statement to follow right after. Reason being: Sounds better if we start with what is a Technology preview feature, market it as an exciting opportunity to try out upcoming feature, and then state the disclaimer that these features aren't supported by RH SLAs.

Reply:
Completely agree, thank you for the feedback :)